cepy/build_cephes.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2017 Gamalon, Inc.
```

Comment:
I agree. It will also make it clearer how to reference the licensing of Cephes.  A possible template for us could be the way that `torch-cephes` has done this https://github.com/deepmind/torch-cephes/blob/master/LICENSE.txt